bentoml/saved_bundle/templates.py

```diff
@@ -82,18 +82,19 @@
 # copy over entrypoint scripts
 COPY docker-entrypoint.sh /usr/local/bin/
 
+# Copy environment.yml, because bundled_pip_dependencies might not exist. This
+# prevent COPY command from failing.
+COPY environment.yml bundled_pip_dependencies*  /bento/bundled_pip_dependencies/
```

Comment:
a Docker conditional COPY hack.

Use `environment.yml` to prevent COPY command fails, and copy all files contain wildcards and matching will be done using Go’s filepath.Match rules, `bundled_pip_dependencies`.

https://docs.docker.com/engine/reference/builder/#copy

Reply:
it was already doing this before this PR

Reply:
moved before the step that run `bentoml-init.sh` script